snapcraft/plugins/copy.py

```diff
@@ -14,6 +14,24 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+"""The copy plugin is useful for assets or other sources with no build system.
+
+This plugin uses the common plugin keywords as well as those for 'sources'
+(though the 'source' keyword is optional). For more information check the
+'plugins' topic for the former and the 'sources' topic for the latter.
+
+Additionally, this plugin uses the following plugin-specific keywords:
+
+    - files:
+      (object)
+      A dictionary of key-value pairs. The key is the current location of the
+      file relative to snapcraft.yaml (unless 'source' is specified, in which
+      case it's relative to the root of the source). The value is where to
+      place the file in-snap, and is relative to the root of the snap. This
+      works like `cp -r <key> <value>`. Note that globbing is supported for the
+      key.
```

Comment:
I expanded on the globbing, though I disagree on backticking key and value since they're referring to multiple items in a dictionary rather than specific variables or commands.